algorithms/aqc/solving_qlsp/solving_qlsp_with_aqc.ipynb

```diff
@@ -0,0 +1,867 @@
+{
```

Comment:
<p>I would write here \ in the complexity part what we did here that was not optimal. e.g suzuki-trotter instead of dyson, non-optimal schedule function, brute-force encoding of the pauli-operators (which scale exponentially)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/688/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>